### ISSUE TITLE
Textile set: link title and image alt text should be optional (alt-click)

### ISSUE THREAD

**Issue by adam-iris** [zsiciarz/django-markitup#18]
Currently, in Textile mode, clicking on the link button prompts for a title, and clicking on the image button prompts for alt text.  Both of these should be optional.  The convention in other places is to define optional components so that they are only included when alt-clicking the toolbar button, that should be the case here.
